### DISCUSSION TITLE
Project specifications

### DISCUSSION THREAD

**Discussion by shai8I** [tanujnotes/Olauncher#413] [General]
Correct me if I'm wrong, but I find that maybe there's some uncertainty in the feature scope, and that contributors are often unaware of the existing discussions?

Also, I would like to ask @tanujnotes about the target audience: are there specific user groups in mind?

**Comment by tanujnotes** [comment]
What do you mean by feature scope? Are you talking about features to be added in the future or existing ones that people may or may not be aware of?

Anyone who finds the app useful is more than welcome to use it. I don't have any specific group in mind.

**Reply by shai8I** [reply]
My apologies, I don't really know enough of project management to explain further. It's just a practice that would help with the direction and anyone who wants to contribute.

I'm just trying to understand better and to possibly help it reach its goals, since it's an app that I actually use.

Every project has some sort of plan, whether fully thought out or not. It's just helpful to communicate it to contributors so they understand.

For example, if it's stated that features are added based on personal use or preference, then contributors would just ask you rather than try to discuss with you or with others. (Not saying that that's the case, just an example.)

BTW, if you need help regarding contributors reading discussions then do let me know.

**Reply by tanujnotes** [reply]
Understood. Olauncher is more or less feature complete so there won't be any major changes going forward. Anyone who wants to contribute can take a look at the open issues or create a new one. My priority is to just maintain this project as long as possible. Let me know if you have any other questions.
